apps/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -88,7 +88,7 @@ export default class PublishAction extends BaseRushAction {
       parameterShortName: '-r',
       key: 'REGISTRY',
       description:
-      `Publishes to a specified NPM registry. If this is specified, it will prevent the current commit will not be ` +
+      `Publishes to a specified npm registry. If this is specified, it will prevent the current commit will not be ` +
```

Comment:
>npm [](start = 32, length = 3)

Wikipedia agrees with me that Isaac doesn't get to make us redesign the rules of English in order to promote his brand:

```
Trademarks that begin with a lowercase letter[edit]
Trademarks that officially begin with a lowercase letter raise several problems because they break the normal capitalization rules of English that proper names are written with initial capital letters wherever they occur in a sentence.

With the exception that immediately follows, trademarks promoted without any capitals are capitalized like any other:
use: I found a Thirtysomething DVD and a pair of Adidas shoes while browsing Craigslist.
avoid: I found a thirtysomething DVD and a pair of adidas shoes while browsing craigslist.
The exception is trademarks that begin with a one-letter lowercase prefix pronounced as a separate letter. These are often not capitalized if the second letter is capitalized, but should otherwise follow normal capitalization rules:
use: He said that eBay is where he bought his iPod.
avoid: He said that EBay is where he bought his IPod.
In such a case, rephrase to avoid beginning sentences with lowercase trademarks:
use: He bought his iPod on eBay.
avoid: eBay is where he bought his iPod.
Not all trademarks with a pronunciation that could have fit this pattern actually do so, and should not be re-styled to conform to it (use NEdit not "nEdit", E-Trade not "eTrade"; Xbox, not "xBox").
```

Reply:
Alternatively, based on this tweet:
[https://twitter.com/npmjs/status/105690425242820608?lang=en](https://twitter.com/npmjs/status/105690425242820608?lang=en)

...I would also be okay with using the symbol from the artist formerly known as Prince.  :-P



---
In reply to: [153675996](https://github.com/Microsoft/web-build-tools/pull/449#discussion_r153675996) [](ancestors = 153675996)